wiki/Ranking_Criteria/en.md

```diff
@@ -61,6 +61,7 @@ Important to understand before using:
   - **...between 3:30 and 4:15**, the lowest difficulty of each included game mode cannot be harder than a Hard.
   - **...between 4:15 and 5:00**, the lowest difficulty of each included game mode cannot be harder than an Insane.
   - **[Break times](/wiki/Beatmap/Break) may be combined with [drain time](/wiki/Beatmap/Drain_time) to meet the above thresholds.** For the highest difficulty, this is limited to at most 30 seconds of break time. This does not apply to difficulties with less than 30 seconds of drain time.
+  - **osu!catch beatmaps are bound by a different set of drain time thresholds defined in the [osu!catch ranking criteria](/wiki/Ranking_Criteria/osu!catch).**
   - **osu!mania beatmaps are bound by a different set of drain time thresholds and other spread rules defined in the [osu!mania ranking criteria](/wiki/Ranking_Criteria/osu!mania).**
```

Comment:
this can be one sentence

Reply:
Not really, because we don't have "other spread rules". Mania has multiple special cases. In catch, we only have different drain time requirements.

Reply:
these drain time thresholds are part of the spread rules anyway. you could write

`  - **osu!catch and osu!mania beatmaps are bound by a different set of spread rules defined in their respective ranking criteria.**`

Reply:
I moved the drain times to their respective game mode RC, so resolving this